### PR TITLE
Add link to checkpointing follow-up article

### PR DESCRIPTION
**Pull request by adrianreber**
This adds a link to the follow-up article "Forensic Container Analysis" to the already existing article "Forensic Container Checkpointing".

/hold

See #39759

### CODE REVIEW

**Comment by sftim** [comment]
/approve

Thanks

File: content/en/blog/_posts/2022-12-05-forensic-container-checkpointing/index.md [hunk 1]
```diff
@@ -207,3 +207,11 @@ and without losing the state of the containers in that Pod.
...
 - Slack: [#sig-node](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
+
+## Further reading
+
+Please see the follow-up article [Forensic container
+analysis][forensic-container-analysis] for details how a container checkpoint
+can be analyzed.
+
+[forensic-container-analysis]: https://kubernetes.io/blog/2023/03/10/forensic-container-analysis/
```
Comment (sftim) [inline content/en/blog/_posts/2022-12-05-forensic-container-checkpointing/index.md:217]:
```suggestion
[forensic-container-analysis]: /blog/2023/03/10/forensic-container-analysis/
```

File: content/en/blog/_posts/2022-12-05-forensic-container-checkpointing/index.md [hunk 2]
```diff
@@ -207,3 +207,11 @@ and without losing the state of the containers in that Pod.
 You can reach SIG Node by several means:
 - Slack: [#sig-node](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
+
+## Further reading
+
+Please see the follow-up article [Forensic container
+analysis][forensic-container-analysis] for details how a container checkpoint
```
Comment (sftim) [inline content/en/blog/_posts/2022-12-05-forensic-container-checkpointing/index.md:214]:
```suggestion
analysis][forensic-container-analysis] for details on how a container checkpoint
```
?

**Review by sftim — COMMENTED** [review]
/lgtm

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**